src/Symfony/Component/HttpClient/CachingHttpClient.php

```diff
@@ -42,7 +42,7 @@ class CachingHttpClient implements HttpClientInterface, ResetInterface
     public function __construct(HttpClientInterface $client, StoreInterface $store, array $defaultOptions = [])
     {
         if (!class_exists(HttpClientKernel::class)) {
-            throw new \LogicException(sprintf('Using "%s" requires that the HttpKernel component version 4.3 or higher is installed, try running "composer require symfony/http-kernel:^4.3".', __CLASS__));
+            throw new \LogicException(sprintf('Using "%s" requires that the HttpKernel component version 4.4 or higher is installed, try running "composer require symfony/http-kernel:^4.4".', __CLASS__));
```

Comment:
While we're at it: I find it strange that the exception message on the 5.4 branch recommends a composer command that would install version 4.4 of the component. Maybe we should drop the constraint from the command and let Composer make the best decision for us?

Reply:
This is indeed something we need to do